### DISCUSSION TITLE
How to add link to a single word in a sentence?

### DISCUSSION THREAD

**Discussion by Sarvom** [fastn-stack/fastn#907] [Q&A] [answered]
How can I add link to specific word in a sentence?

**Comment by gsalunke** [comment, marked as answer]
@Sarvom 
Please, use markdown syntax for inline links.

E.g.
Liking fastn? [click here](https://fastn.com/)
use:
```ftd
I am a normal [link](url-of-link)
Liking fastn? [click here](https://fastn.com/)
```

**Reply by amitu** [reply]
@gsalunke I have updated your comment. It's markdown not markup. Markup syntax is totally different thing.
